RosNet/RosReader/Header.cs

```diff
@@ -14,7 +14,7 @@ namespace RosNet.RosReader
     public static class Header
```

Comment:
Remove static from this class, and extract the public members to an interface. Then you can revert the internals to public again, and just have the class be internal. That will be a lot cleaner that this public class with so many internal members

Reply:
So everything in the Interface would then be public, and everything not in the interface would default to internal, like the class?

Reply:
This entire class should rather be internal? Or rather each method should be a private method on `RosReader`. I don't see how anyone is going to re-use this. No reason to use interface as you don't need to abstract over different implementations

Reply:
I just saw that there are no usefull members in this class that should be exposed publicly, so then I agree with just making the whole thing internal

Reply:
I think moving it into the main Reader class would be the most fitting here, if we also want to remove as many static classes and methods as possible